### DISCUSSION TITLE
Not able to get the gender Information as an entity on a audio clip.

### DISCUSSION THREAD

**Discussion by somanshusingh1** [deepgram/community#159] [General help]
I'm trying to gather the gender information of the speakers in an audio clip. I'm currently accessing the API.
I used this way:
curl \
  --request POST \
  --header 'Authorization: Token YOUR_DEEPGRAM_API_KEY' \
  --header 'Content-Type: audio/wav' \
  --data-binary @youraudio.wav \
  --url 'https://api.deepgram.com/v1/listen?detect_entities=true&punctuate=true'

Not getting gender data. Please help if I'm missing something

**Comment by lukeocodes** [comment]
We cannot identify gender from who is speaking in an audio clip, but we can output gender as part of our [entity detection](https://developers.deepgram.com/docs/detect-entities#identifiable-entities) feature. This would output a gender when a word is used to identify one. Like 'she did', 'female', 'it's hers'. Could you provide the resulting transcript or even the audio file?
